### PR TITLE
Add Improve Digital adapter to Prebid

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This includes 'Improve Digital's adapter in Prebid auctions in the UK and International editions, for testing.
The placement ID values were supplied by Spencer.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js [hunk 1]
```diff
@@ -57,30 +59,118 @@ const getIndexSiteId = (): string => {
...
+const contains = (sizes: PrebidSize[], size: PrebidSize): boolean =>
+    Boolean(sizes.find(s => s[0] === size[0] && s[1] === size[1]));
+
+const getImprovePlacementId = (sizes: PrebidSize[]): number => {
+    switch (config.page.edition) {
+        case 'UK':
+            switch (getBreakpointKey()) {
+                case 'D':
+                    if (
+                        contains(sizes, [300, 250]) ||
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js]: you might want to add a `const` for this boolean logic, and another for the banner sizes.

File: static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js [hunk 2]
```diff
@@ -57,30 +59,118 @@ const getIndexSiteId = (): string => {
     return site ? site.id : '';
 };
 
+const contains = (sizes: PrebidSize[], size: PrebidSize): boolean =>
+    Boolean(sizes.find(s => s[0] === size[0] && s[1] === size[1]));
```
Comment (LATaylor-guardian) [inline static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js]: How can you be sure that the array is not empty and has two or more elements?

**Review by rich-nguyen — APPROVED** [review]

**Comment by LATaylor-guardian** [comment]
👍

File: static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js (hunk 2 above)
Comment (kelvin-chappell) [reply static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js]: If it's empty the result should be false.
And a `PrebidSize` is defined as `[number, number]` so should always have exactly two elements.

**Pull request merged**